app/models/chargeback.rb

```diff
@@ -64,7 +64,7 @@ def self.build_results_for_report_chargeback(options)
         data[key] ||= new(options, consumption, region.region)
 
         chargeback_rates = data[key]["chargeback_rates"].split(', ') + rates_to_apply.collect(&:description)
-        data[key]["chargeback_rates"] = chargeback_rates.uniq.join(', ')
+        data[key]["chargeback_rates"] = _(chargeback_rates.uniq.join(', '))
```

Comment:
should the "Chargeback Rates" column be translated or should I remove this change? @lpichler @mzazrivec @skateman

Reply:
This change won't do anything really. I'm quite certain that whatever is the result of `chargeback_rates.uniq.join(', ')`, it will not be in our catalogs. It needs to go away.

Reply:
`chargeback_rates.uniq.join(', ')` is the "Default" string from the above image, I just have no idea where it actually comes from

Reply:
`chargeback_rates.uniq.join(', ')` is the "Default"  and can be also  "Default, Rate 2,...". If it be need we need to apply `_()` on each item of array `chargeback_rates`.

@mzazrivec 
We display `ChargebackRate#description` on chargeback report (see desc. chargeback rate) 
`ChargebackRate#description`  is editable only when user creates new chargeback rates.
We have 2 or 3 default ChargebackRate which are not editable and there are seeded and question is if we should mark `ChargebackRate#description` of chargeback rates which are default on report for translations  ?

Reply:
@lpichler no, we don't try to translate strings which come from database, since we consider them a user-defined values (even if in few cases those user-defined values are pre-seeded from installation).

Reply:
so remove this then?

Reply:
@DavidResende0 Yes please.

Reply:
done